presto-main/src/main/java/com/facebook/presto/operator/WindowOperator.java

```diff
@@ -192,6 +192,7 @@ public OperatorFactory duplicate()
     private final PageBuilder pageBuilder;
 
     private final SingleDriverWindowInfoBuilder windowInfo;
+    private SingleDriverWindowInfo singleDriverWindowInfo;
```

Comment:
Merge this with the commit that introduced it. Also, I don't see what you're referring to in HashAggregationOperator could you be more specific?